xml/xml_load_world.cpp

```
@@ -9,6 +9,317 @@
 #include "..\mainfrm.h"
 #include "..\dialogs\ProgDlg.h"
 
+template <class T>
+static T * SimulateXMLLoadRollback (
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)